### PR TITLE
19). Add topic files 47 to 50

### PR DESCRIPTION
**Pull request by jameshkramer**
@mairaw @BillWagner
This PR is for user story #1576037.

It adds four article files, makes corresponding deletions from the C# and VB folders, and adds redirection entries. There are also some fixes to past articles.

Please merge https://github.com/dotnet/docs/pull/17675 before merging this.

cc: @tfosmark @marteeleigh

Contributes to #4728

### CODE REVIEW

**Comment by mairaw** [comment]
@jameshkramer do you wanna resolve the conflicts here now that I merged the previous PR?

**Comment by mairaw** [comment]
You need to actually resolve the conflicts. You can do via the UI (https://github.com/dotnet/docs/pull/17688/conflicts) or locally on your computer.

File: docs/standard/linq/query-xml-trees-overview.md [hunk 1]
```diff
@@ -13,11 +13,11 @@ For background information about LINQ queries, see:
 - [Introduction to LINQ Queries (C#)](../../csharp/programming-guide/concepts/linq/introduction-to-linq-queries.md)
 - [Writing Your First LINQ Query (Visual Basic)](../../visual-basic/programming-guide/concepts/linq/writing-your-first-linq-query.md)
 
-## In this section
+## In This Section
```
Comment (BillWagner) [inline docs/standard/linq/query-xml-trees-overview.md:16]: This heading should have remained sentence cased:

```suggestion
## In this section
```

**Review by BillWagner — CHANGES_REQUESTED** [review]
I noticed one item to fix. After that, I'm good.

**Comment by Thraka** [comment]
Lots of missing links to fix in the build report.

**Comment by jameshkramer** [comment]
@Thraka The cause for most, perhaps all, of the broken links is that only about 1/3 of the articles for the LINQ to XML section have been added so far.  The links will be fixed over time as articles are added. At the end we will attend to any remaining ones.

**Comment by jameshkramer** [comment]
@BillWagner thanks, I committed your change.

File: docs/standard/linq/find-elements-specific-attribute.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+---
+title: "How to find elements with a specific attribute (XPath-LINQ to XML) (C#)"
```
Comment (mairaw) [inline docs/standard/linq/find-elements-specific-attribute.md:2]:
```suggestion
title: "How to find elements with a specific attribute - XPath-LINQ to XML"
```

File: docs/standard/linq/find-elements-specific-attribute.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+ms.date: 07/20/2015
+ms.assetid: daed00dd-923a-43be-8a90-eee406f6f574
+---
+# How to find elements with a specific attribute (XPath-LINQ to XML) (C#)
```
Comment (mairaw) [inline docs/standard/linq/find-elements-specific-attribute.md:6]:
```suggestion
# How to find elements with a specific attribute in C# (XPath-LINQ to XML)
```

**Review by mairaw — COMMENTED** [review]
Left a few comments but LGTM

**Comment by jameshkramer** [comment]
I deleted the file in which Maira found problems. Solved! I had inadvertently included the file because it has a very similar name to another. It had never been properly edited.  It is scheduled to be added later.

**Comment by jameshkramer** [comment]
I did commit Bill's change. I don't know why GitHub seems to indicate that there is more to do.

**Review by BillWagner — APPROVED** [review]

**Comment by mairaw** [comment]
He just needed to re-review @jameshkramer. Merging now.

**Pull request merged**